wiki/Contests/Kyoku/2024/en.md

```diff
@@ -17,12 +17,18 @@
 
 | Ranking | Prizes |
 | :-- | :-- |
-| Winning team | $100 USD + profile badge (pending) + [contest points](/wiki/Contests/Contest_points) |
+| Winning team | $100 USD + profile badge + 2 [contest points](/wiki/Contests/Contest_points) |
+| 2nd place | 1 contest point |
+
+![Kyoku 2024 winner's badge](img/badge.png)
```

Comment:
why not use the assets.ppy.sh link for this one?

Reply:
because most tournaments and contests don't do that